js/utils/follow.js

```diff
@@ -0,0 +1,49 @@
+import $ from 'jquery';
+import app from '../app';
+import Dialog from '../views/modals/Dialog';
+
+export default {
+
+  followedByYou(guid) {
+    return app.ownFollowing.get(guid) !== undefined;
```

Comment:
If you wanted, this could be written more concisely as `!!app.ownFollowing.get(guid)`.